backend/codesirius/problems/serializers/tag.py

```
@@ -0,0 +1,41 @@
+    name = serializers.CharField(
+        max_length=50,
+        required=True,
+        error_messages={
+            "required": "The name field is required.",
+            "blank": "The name field must not be blank.",
+            "null": "The name field must not be null.",
+            "max_length": "The name field must not exceed 50 characters.",
+        },
+        validators=[
+            UniqueValidator(
+                queryset=Tag.objects.all(), message="Tag with this name already exists."
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)